index.bs

```diff
@@ -7835,8 +7781,7 @@ that correspond to the union’s [=member types=].
             |V| to that callback function type.
         1.  If |types| includes {{object}}, then return the IDL value
             that is a reference to the object |V|.
-    1.  If |V| is any kind of object except for
-        a native <emu-val>RegExp</emu-val> object, then:
+    1.  If |V| is any kind of object, then:
```

Comment:
I have no problem with changing all this stuff to use Type(V).  In fact, I think it would be a good idea.  Doing it in a separate PR from this one is probably best.